packages/typespec-go/.scripts/tspcompile.js

```diff
@@ -117,16 +117,16 @@ function should_generate(name) {
 }
 
 const armcodesigning = pkgRoot + 'test/tsp/CodeSigning.Management';
-generate('armcodesigning', armcodesigning, 'test/armcodesigning', ['remove-unreferenced-types=true']);
+generate('armcodesigning', armcodesigning, 'test/armcodesigning');
 
 const armapicenter = pkgRoot +  'test/tsp/ApiCenter.Management';
-generate('armapicenter', armapicenter, 'test/armapicenter', ['remove-unreferenced-types=true']);
+generate('armapicenter', armapicenter, 'test/armapicenter');
 
 const armlargeinstance = pkgRoot + 'test/tsp/AzureLargeInstance.Management';
-generate('armlargeinstance', armlargeinstance, 'test/armlargeinstance', ['stutter=AzureLargeInstance', 'remove-unreferenced-types=true']);
+generate('armlargeinstance', armlargeinstance, 'test/armlargeinstance', ['stutter=AzureLargeInstance']);
 
 const armdatabasewatcher = pkgRoot + `test/tsp/DatabaseWatcher.Management`;
-generate('armdatabasewatcher', armdatabasewatcher, 'test/armdatabasewatcher');
+generate('armdatabasewatcher', armdatabasewatcher, 'test/armdatabasewatcher', ['remove-unreferenced-types=false']);
```

Comment:
Do we now get `--option="...remove-unreferenced-types=false" --option="...remove-unreferenced-types=true"` in the command line?

Reply:
Yes. Last one appears to win. It was the simplest solution instead of wiring up stuff to change the default value.

Reply:
Fair enough. I'll improve the script.

Reply:
Done.